SomerenUI/SomerenUI.cs

```diff
@@ -77,6 +77,17 @@ private void ShowActivitiesPanel()
             DisplayDataInListView(listViewActivities, data, CreateActivityListViewItem);
         }
 
+        private void ShowActivitiesSupervisorsPanel()
```

Comment:
In my opinion, it makes sense for this (and the other similar uses of it) to be named ShowActivitySupervisorsPanel. Just making the change from Activities to Activity. I believe that would be the proper way to name it